src/components/Main/Main.tsx

```diff
@@ -22,12 +24,12 @@ const Main: React.FC<MainProps> = ({
   closeModal,
   movies,
   openModal,
+  setOpenedFilmId,
 }) => {
   const [selectedGenre, setGenre] = useState(DefaultFilters.defaultGenre);
-  const [sortBy, setSort] = useState("");
+  const [sortBy, setSort] = useState(DefaultFilters.defaultSort);
 
   const onChangeGenre = (event: React.MouseEvent) => {
-    setSort("");
     setGenre((prevGenre) =>
       (event.target as HTMLInputElement).id
         ? (event.target as HTMLInputElement).id
```

Comment:
correct`event.target as HTMLInputElement).id || prevGenre`
i repeated this comment probably a 3rd time